messaging/kafka-avro-reactive-messaging/src/main/resources/application.properties

```diff
@@ -1,19 +1,18 @@
-# Configuration file - Quarkus profile: Strimzi
-kafka.bootstrap.servers=kafka-broker-1:9092
-# TODO: Disable health check because of https://github.com/quarkusio/quarkus/issues/15464
-quarkus.reactive-messaging.health.enabled=false
-mp.messaging.connector.smallrye-kafka.apicurio.registry.url=${kafka.registry.url}
-mp.messaging.outgoing.source-stock-price.connector=smallrye-kafka
-mp.messaging.outgoing.source-stock-price.topic=stock-price
-mp.messaging.outgoing.source-stock-price.value.serializer=io.apicurio.registry.utils.serde.AvroKafkaSerializer
-mp.messaging.outgoing.source-stock-price.apicurio.registry.artifact-id=io.apicurio.registry.utils.serde.strategy.SimpleTopicIdStrategy
-mp.messaging.outgoing.source-stock-price.apicurio.registry.global-id=io.apicurio.registry.utils.serde.strategy.GetOrCreateIdStrategy
-mp.messaging.outgoing.source-stock-price.apicurio.registry.avro-datum-provider=io.apicurio.registry.utils.serde.avro.ReflectAvroDatumProvider
-mp.messaging.incoming.channel-stock-price.connector=smallrye-kafka
-mp.messaging.incoming.channel-stock-price.topic=stock-price
-mp.messaging.incoming.channel-stock-price.specific.avro.reader=true
-mp.messaging.incoming.channel-stock-price.value.deserializer=io.apicurio.registry.utils.serde.AvroKafkaDeserializer
-mp.messaging.incoming.channel-stock-price.auto.offset.reset=earliest
-mp.messaging.incoming.channel-stock-price.enable.auto.commit=true
-mp.messaging.incoming.channel-stock-price.apicurio.registry.avro-datum-provider=io.apicurio.registry.utils.serde.avro.ReflectAvroDatumProvider
-quarkus.openshift.expose=true
+#Configuration file - Quarkus profile: Strimzi
+%dev.kafka.bootstrap.servers=localhost:9092
```

Comment:
Why did you move these properties into the `dev` profile?

Reply:
because those properties were used by our test and are not compatible with Confluent scenarios. On the other hand, I don't want to lose the ability to launch this app locally (manually).